tools/reviewer/job/tasks/ReviewerMetadataUpdaterTask.java

```diff
@@ -153,5 +163,148 @@ public void run() {
   public Boolean shouldRun() {
     return !lock.isLocked();
   }
+
+  public ReviewerConfig getReviewerConfig(String filePath) throws IOException {
+    ReviewerConfig reviewerConfig =
+        (ReviewerConfig) fileUtils.readPrototxt(filePath, ReviewerConfig.newBuilder());
+    return reviewerConfig;
+  }
+
+  public String getStartupOsReviewerConfigPath() {
+    String localRegistryFilePath =
+        fileUtils.joinPaths(fileUtils.getCurrentWorkingDirectory(), "reviewer_config.prototxt");
+    return localRegistryFilePath;
+  }
+
+  public String getHasadnaReviewerConfigPath() {
+    String targetDirectory = fileUtils.expandHomeDirectory("~/hasadna");
+    String localHasadnaRegistryFilePath =
+        fileUtils.joinPaths(targetDirectory, "reviewer_config.prototxt");
+    return localHasadnaRegistryFilePath;
+  }
+
+  private User getUser(ReviewerConfig reviewerConfig, String userId) {
+    for (User user : reviewerConfig.getUserList()) {
+      if (user.getId().equals(userId)) {
+        return user;
+      }
+    }
+    return User.getDefaultInstance();
+  }
+
+  public void compareReviewerConfigData(
```

Comment:
I think the method should return `mergedReviewerConfig`.

Reply:
Not `mergedReviewerConfig.build()`?

Reply:
The method should return `ReviewerConfig` message, not  `ReviewerConfig.Builder`. 
`mergedReviewerConfig.build()` is correct.